tests/accuracy_tests/sim_for_alphapeel_accu_test/sim_for_accu.R

```diff
@@ -0,0 +1,248 @@
+rm(list = ls())
+library(AlphaSimR)
+library(Rlab)
+
+nLoci <- 1000
+nInd <- 1000
+nGen <- 5
+nChr <- 1
+nsegSites <- nLoci/nChr
+genoError <- 0.001
+seqError <- 0.01
+nSelect <- 100
+
+# runMacs for generic species
+founderGenomes <- runMacs(nInd = (nInd / nGen),
+                          nChr = nChr,
+                          segSites = nsegSites,
+                          species = "GENERIC")
+SP <- SimParam$new(founderGenomes)
+SP$setSexes("yes_sys")
+SP$setTrackPed(TRUE)
+SP$setTrackRec(TRUE)
+basePop <- newPop(founderGenomes)
+
+currentPop <- basePop
+Parents <- c()
+for (generation in 1:(nGen - 1)) {
+  # add selections, pick 100 parents randomly, make the parents high density, everyone in gen5 low density
+  newParents <- selectInd(currentPop, nInd = nSelect, use = "rand")
+  Parents <- c(Parents, getPed(newParents)[, 1])
+  newPop <- randCross(currentPop, nCrosses = nInd(currentPop))
+  currentPop <- newPop
+  popList <- list(basePop, currentPop)
```

Comment:
@XingerTang this won't work - we keep adding `basePop` to `popList` while we should only add it once

Reply:
@gregorgorjanc `basePop` here is not a very appropriate variable name, as it actually represents the total population by far, it should be replaced by something like `totalPop`. However, this piece of code should work, as each time the `popList` is assigned instead of being appended to the combined list of `basePop` and `currentPop`, so only `currentPop` is added each iteration. `basePop` would always contain one copy of each generation only.

Reply:
@gregorgorjanc I've read your version of the code, and I will make sure to name the variables more carefully next time.